actions/pacstrap_action.go

```diff
@@ -71,6 +72,15 @@ func NewPacstrapAction() *PacstrapAction {
 	return &d
 }
 
+func (d *PacstrapAction) PreMachine(context *debos.DebosContext, m *fakemachine.Machine, args *[]string) error {
```

Comment:
Nit: add a inline comment - "Use host keyring to speed up resolving public keys" or alike